### PR TITLE
Update packages related to latest libsonatareport changes

### PR DESCRIPTION
**Pull request by iomaganaris**

### CODE REVIEW

**Review by alkino — APPROVED** [review]

File: bluebrain/repo-bluebrain/packages/libsonata-report/package.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class LibsonataReport(CMakePackage):
     git = "https://github.com/BlueBrain/libsonatareport.git"
 
     version('develop', branch='master', submodules=False, get_full_repo=True)
+    version('1.0.0.20220218', commit='905641', submodules=False)
```
Comment (olupton) [inline bluebrain/repo-bluebrain/packages/libsonata-report/package.py:20]: Shouldn't this be a much more significant version number change, given that it breaks API?

Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/libsonata-report/package.py:20]: ☝️

Comment (iomaganaris) [reply bluebrain/repo-bluebrain/packages/libsonata-report/package.py:20]: We discussed this with Jorge and he said that he's going to create a new version after adding some bigger changes

Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/libsonata-report/package.py:20]: Maybe push more smaller releases in the future, there's a whole bunch of dated versions, _and that should not be the default behavior_.

**Review by matz-e — APPROVED** [review]

**Pull request merged**